lib/assets/javascripts/turbograft/turbolinks.coffee

```diff
@@ -155,6 +157,18 @@ class window.Turbolinks
 
     return matchingNodes
 
+  anyAutofocusElement = (nodes) ->
+    for node in nodes
+      if node.querySelectorAll('input[autofocus], textarea[autofocus]').length > 0
+        return true
```

Comment:
so I forget my coffeescript, does this return from function or is it returning `true` to the comprehension of `for node in nodes`?